config.go

```diff
@@ -674,6 +676,19 @@ func loadConfig() (*config, error) {
 		}
 	}
 
+	if cfg.Tor.WatchtowerKeyPath == "" {
+		switch {
+		case cfg.Tor.V2:
+			cfg.Tor.WatchtowerKeyPath = filepath.Join(
+				cfg.Watchtower.TowerDir, defaultTorV2PrivateKeyFilename,
+			)
+		case cfg.Tor.V3:
+			cfg.Tor.WatchtowerKeyPath = filepath.Join(
+				cfg.Watchtower.TowerDir, defaultTorV3PrivateKeyFilename,
+			)
```

Comment:
Nit: could extract this into a function and reuse for lnd's as well.

Reply:
Could you be more specific? It (`WatchtowerKeyPath`) is not in `lnd.go`. Or do you mean making the initialization of `WatchtowerKeyPath` and `PrivateKeyPath` a separate function? Because I don't know if that would save any lines of code? I could add a comment though for clarity, wyt?